handlers/index.go

```diff
@@ -6,40 +6,64 @@ import (
 	"eth2-exporter/templates"
 	"eth2-exporter/types"
 	"eth2-exporter/utils"
-	"html/template"
 	"net/http"
 )
 
-var indexTemplateFiles = []string{
-	"layout.html",
-	"index/index.html",
-	"index/depositProgress.html",
-	"index/depositChart.html",
-	"index/genesis.html",
-	"index/hero.html",
-	"index/networkStats.html",
-	"index/participationWarning.html",
-	"index/postGenesis.html",
-	"index/preGenesis.html",
-	"index/recentBlocks.html",
-	"index/recentEpochs.html",
-	"index/genesisCountdown.html",
-	"index/depositDistribution.html",
-	"components/banner.html",
-	"svg/bricks.html",
-	"svg/professor.html",
-	"svg/timeline.html",
-	"components/rocket.html",
-	"slotViz.html",
-}
-
-var indexTemplate = template.Must(template.New("index").Funcs(utils.GetTemplateFuncs()).ParseFS(templates.Files,
-	indexTemplateFiles...,
-))
-
 // Index will return the main "index" page using a go template
 func Index(w http.ResponseWriter, r *http.Request) {
 
+	/* var indexTemplateFiles = []string{
```

Comment:
Generally I am not a fan of leaving unused code within the sources. I would remove that (and some other unused stuff).

Reply:
Removed in new commit